app/assets/javascripts/application.js

```diff
@@ -11,6 +11,8 @@
 // about supported directives.
 //
 //= require rails-ujs
+//= require jquery
+//= require jquery_ujs
```

Comment:
You don't have to add jquery_ujs here, as you already have rails_ujs ([Ref](https://gorails.com/forum/do-i-need-rails-ujs-and-jquery_ujs))